scripts/updates/setup-updates.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Set the directory to the script's directory
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)